### PR TITLE
Font size changed for large screen

### LINKED ISSUE #52: Font too big on large screens
Compress everything and make the layout more friendly for bigger screens by increasing the column size of the grids. This should remove a bunch of `lg:...` styles, most notably text size and padding.

### PR DESCRIPTION
**Pull request by RajeevDash007**
I deleted all the lg related classes to make the font size smaller for large screens. Fixes #52.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -30,27 +30,27 @@ body {
 }
 
 h1 {
-  @apply text-6xl lg:text-7xl;
+  @apply text-6xl ;
```
Comment (jofas) [inline src/style.css:33]: Just a tiny nit, but could you remove the whitespace between the class name and the semicolon?

```suggestion
  @apply text-6xl;
```

File: src/style.css [hunk 2]
```diff
@@ -30,27 +30,27 @@ body {
...
 }
 
 h2 {
-  @apply font-bold text-4xl lg:text-6xl;
+  @apply font-bold text-4xl ;
```
Comment (jofas) [inline src/style.css:37]:
```suggestion
  @apply font-bold text-4xl;
```

File: src/style.css [hunk 3]
```diff
@@ -30,27 +30,27 @@ body {
...
 }
 
 p {
-  @apply font-mono text-base/loose lg:text-xl/loose;
+  @apply font-mono text-base/loose ;
```
Comment (jofas) [inline src/style.css:41]:
```suggestion
  @apply font-mono text-base/loose;
```

**Review by jofas — COMMENTED** [review]
Thanks for the PR! Looks good to me. Just one small issue with the format and this is ready to be merged :slightly_smiling_face:

**Comment by RajeevDash007** [comment]
I have eliminated the white spaces

**Comment by jofas** [comment]
Thanks a lot.

**Pull request merged**